ansible/roles/test/files/ptftests/copp_tests.py

```diff
@@ -32,14 +32,15 @@ class ControlPlaneBaseTest(BaseTest):
     PPS_LIMIT_MAX = PPS_LIMIT * 1.1
     NO_POLICER_LIMIT = PPS_LIMIT * 1.4
     PKT_TX_COUNT = 100000
-    PKT_RX_LIMIT = PKT_TX_COUNT * 0.90
     TASK_TIMEOUT = 300 # Wait up to 5 minutes for tasks to complete
 
     def __init__(self):
         BaseTest.__init__(self)
         self.log_fp = open('/tmp/copp.log', 'a')
         test_params = testutils.test_params_get()
         self.verbose = 'verbose' in test_params and test_params['verbose']
+        self.pkt_tx_count = test_params.get('pkt_tx_count', self.PKT_TX_COUNT)
```

Comment:
Here you will read this None value (probably as string), because you initialized it as default in ansible code.